### PR TITLE
[CLOUDTRUST-4933] Remove events, statistics and export package + remo…

### PR DESCRIPTION
**Pull request by basbeu**
…ve audit event mariadb

### CODE REVIEW

File: pkg/events/authorization.go [hunk 1]
```diff
@@ -1,88 +0,0 @@
...
-		return &authorizationComponentMW{
-			authManager: authorizationManager,
-			logger:      logger,
-			next:        next,
-		}
-	}
-}
-
-func (c *authorizationComponentMW) GetActions(ctx context.Context) ([]api.ActionRepresentation, error) {
-	var action = security.EVGetActions.String()
```
Comment (fperot74) [inline pkg/events/authorization.go:31]: Actions might be removed from common-service? or this code was already moved in another service?

File: pkg/statistics/authorization.go [hunk 2]
```diff
@@ -41,16 +41,6 @@ func (c *authorizationComponentMW) GetActions(ctx context.Context) ([]api.Action
 	return c.next.GetActions(ctx)
 }
 
-func (c *authorizationComponentMW) GetStatistics(ctx context.Context, realm string) (api.StatisticsRepresentation, error) {
-	var action = security.STGetStatistics.String()
```
Comment (fperot74) [inline pkg/statistics/authorization.go:45]: Same for these actions...

**Review by fperot74 — APPROVED** [review]

File: pkg/events/authorization.go (hunk 1 above)
Comment (basbeu) [reply pkg/events/authorization.go:31]: Used in event service

File: pkg/statistics/authorization.go (hunk 2 above)
Comment (basbeu) [reply pkg/statistics/authorization.go:45]: Same here

**Pull request merged**